src/main/java/com/xceptance/xlt/api/engine/DataLogger.java

```diff
@@ -0,0 +1,34 @@
+package com.xceptance.xlt.api.engine;
+
+/**
+ * The {@link DataLogger} logs custom data for a specific scope to a log file, from where they may be read again 
+ * during test report generation. The {@link DataLogger} instance responsible for a certain test user may be 
+ * obtained from the current session's data manager object via {@link Session#getDataManager()#dataLogger(String scope)}.
+ */
+public interface DataLogger
+{
+    /**
+     * Adds a header to the log file. 
+     * This only makes sense for csv files and only if no data is in the file yet, 
+     * so the header may not be written if this is not the case.
+     * @param header
+     */
+    public void setHeader(String header);
+    //TODO what about format? is this freetext string or do we expect csv, i.e. a number of strings seperated 
+    //by a fixed delimiter defined seperately?
```

Comment:
I believe it is free text because we do not impose a format for the data lines either.